pkg/operation/botanist/etcd.go

```diff
@@ -48,6 +50,20 @@ func (b *Botanist) DefaultEtcd(role string, class etcd.Class) (etcd.Interface, e
 		return nil, err
 	}
 
+	// Extract gardenlet config
+	var gardenletConfig *configv1alpha1.GardenletConfiguration
+	if b.ManagedSeed != nil {
+		gardenletConfig, err = helper.ExtractGardenletConfig(b.ManagedSeed)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
Hm, can't we simply use `b.Config`? This is also more accurate AFAIK because the Gardenlet can be started without a `ManagedSeed`. (/cc @rfranzke)

Reply:
Good catch 👀

Reply:
Done